src/main/java/com/commercetools/sync/products/ProductSync.java

```diff
@@ -85,7 +87,7 @@ public ProductSync(@Nonnull final ProductSyncOptions productSyncOptions) {
             new ProductTypeServiceImpl(productSyncOptions),
             new CategoryServiceImpl(CategorySyncOptionsBuilder.of(productSyncOptions.getCtpClient()).build()),
             new TypeServiceImpl(productSyncOptions),
-            new ChannelServiceImpl(productSyncOptions),
+            new ChannelServiceImpl(productSyncOptions, Collections.singleton(ChannelRole.PRODUCT_DISTRIBUTION)),
```

Comment:
I wonder why not changing the default constructor instead of changing it here, as described on the issue before?

https://github.com/commercetools/commercetools-sync-java/blob/7baae24007cbb49afe7a6345bfee535765f3ac7b/src/main/java/com/commercetools/sync/services/impl/ChannelServiceImpl.java#L35-L38

Reply:
Also codecov report is pointing the same place:

<img width="1139" alt="Screenshot 2020-04-21 at 09 14 47" src="https://user-images.githubusercontent.com/3469524/79836593-9f668400-83b0-11ea-9201-bf4e3b4760e1.png">

Reply:
I think enforcing the user to aware about the mandatory fields is better rather setting those values inside the constructor. With the current approach, user knows that is not possible to instantiate a ChannelService without knowing channel roles.

Reply:
abec68c